### PR TITLE
Add section to tracing task to cover sampling.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 1]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio is configured by default to capture all traces. For every time
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Istio captures all traces by default. Every time

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 2]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+you access `/productpage` you should see a corresponding trace in the
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: you access `/productpage`, you see a corresponding ...

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 3]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+Jaeger dashboard. This is suitable for a test or low traffic mesh. For
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: This path is suitable for a test or low-traffic mesh.

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 4]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+a high traffic mesh you may wish to lower the trace sampling
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: a high traffic mesh, you can lower the trace sampling

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 5]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+percentage. This can be set in two ways:
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: percentage in one of two ways:

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 6]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+
+* On mesh setup, you may use the Helm option `pilot.traceSampling` to
+  set the percentage of trace sampling you want. See the
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: * During the mesh setup, use the `pilot.traceSampling` Helm option to
  set the percentage of trace sampling.

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 7]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+  [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
+  details on setting options.
+* In a running mesh, you may edit the `istio-pilot` deployment and
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: In a running mesh, edit the `istio-pilot` deployment and

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 8]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+  change the environment variable by doing the below.
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: change the environment variable with the following steps:

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 9]
```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+
+  1. Run `kubectl -n istio-system edit deploy istio-pilot`. This will
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: That command doesn't just open the text editor. If I am not mistaken, it opens the `istio-pilot configuration file on your text editor. Please rewrite the text to reflect the exact behavior the command causes. I also recommend placing the command within the {{< text bash >}}{{< /text >}} tags.

Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated, had to move above the command to preserve the numbered list.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 8 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 7 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 6 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 5 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 4 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 3 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Not path, but sampling rate, updated to say that.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 2 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Updated.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 1 above)
Comment (jeffmendoza) [reply content/docs/tasks/telemetry/distributed-tracing/index.md]: Made this more clear, captures traces for all requests.

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 10]
```diff
@@ -139,25 +139,31 @@ When you make downstream calls in your applications, make sure to include these
...
   [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
   details on setting options.
-* In a running mesh, you may edit the `istio-pilot` deployment and
-  change the environment variable by doing the below.
+* In a running mesh, edit the `istio-pilot` deployment and
+  change the environment variable with the following steps.
 
-  1. Run `kubectl -n istio-system edit deploy istio-pilot`. This will
-     open your text editor.
+  1. Run the below command. It will open your text editor with the deployment configuration
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Verb tense congruence: avoid past and future tenses: To fix, I suggest:
1. To open your text editor with the deployment configuration file loaded, run the following command:

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 11]
```diff
@@ -139,25 +139,31 @@ When you make downstream calls in your applications, make sure to include these
...
+percentage in one of two ways:
+
+* During the mesh setup, use the Helm option `pilot.traceSampling` to
+  set the percentage of trace sampling. See the
   [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
   details on setting options.
-* In a running mesh, you may edit the `istio-pilot` deployment and
-  change the environment variable by doing the below.
+* In a running mesh, edit the `istio-pilot` deployment and
+  change the environment variable with the following steps.
```
Comment (rcaballeromx) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Punctuation: Use colons to introduce lists.
... with the following steps:

**Comment by jeffmendoza** [comment]
@rcaballeromx Updated with the last 2 comments. Thanks for the help!

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 12]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Lowercase s

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 13]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+
+Istio captures a trace for all requests by default. Every time you
+access `/productpage` you see a corresponding trace in the Jaeger
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: You need to preface this with something like:

When using the Bookinfo sample application for example,

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 14]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+percentage in one of two ways:
+
+* During the mesh setup, use the Helm option `pilot.traceSampling` to
+  set the percentage of trace sampling. See the
+  [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
+  details on setting options.
+* In a running mesh, edit the `istio-pilot` deployment and
+  change the environment variable with the following steps:
+
+  1. To open your text editor with the deployment configuration file
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Needs to be indented by 4 spaces.

File: content/docs/tasks/telemetry/distributed-tracing/index.md (hunk 12 above)
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: Lowercase s

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 15]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio captures a trace for all requests by default. Every time you
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: You need to preface the second sentence with something like:

For example, when using the Bookinfo sample application, ...

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 16]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+  set the percentage of trace sampling. See the
+  [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
+  details on setting options.
+* In a running mesh, edit the `istio-pilot` deployment and
+  change the environment variable with the following steps:
+
+  1. To open your text editor with the deployment configuration file
+     loaded, run the following command:
+
+     {{< text bash >}}
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: This needs to be indented a multiple of 4, and the content should be lined up with the {{

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 17]
```diff
@@ -137,6 +137,35 @@ public Response bookReviewsById(@PathParam("productId") int productId,
...
+    $ kubectl -n istio-system edit deploy istio-pilot
+     {{< /text >}}
+
+  1. Find the `PILOT_TRACE_SAMPLING` environment variable, and change
+     the `value:` to your desired percentage.
+
+  1. Save and exit your text editor.
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: No need for this step.

**Comment by jeffmendoza** [comment]
@geeknoid Updated

**Pull request merged**